components/dialogs/DisabledDialog.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <v-row justify="center">
+        <v-dialog v-model="dialog" disabled=true max-width="290">
```

Comment:
disabled=true -> disabled
We don't need explicitly specify 'true' in this case

Reply:
Fixed